src/PantherTestCaseTrait.php

```diff
@@ -161,12 +167,18 @@ protected static function createPantherClient(array $options = [], array $kernel
             }
         }
 
+        // Only allow browser arguments if a browser has been explicitly specified
+        if (\is_array($browserArguments) && !\array_key_exists('browser', $options)) {
+            @trigger_error('Specifying $browserArguments requires that you specifically set the browser in $options. $browserArguments will be ignored.', \E_USER_WARNING);
```

Comment:
I'm now triggering this warning if a user failed to specify a browser while adding browser arguments.